mungegithub/mungers/release-note-label_test.go

```diff
@@ -38,107 +39,120 @@ var (
 func TestReleaseNoteLabel(t *testing.T) {
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
+	var (
+		comment1 = "Release note process has not been followed"
+		comment2 = "The 'parent' PR of a cherry-pick PR must have one of the"
+	)
+
 	tests := []struct {
-		name        string
-		issue       *github.Issue
-		body        string
-		branch      string
-		secondIssue *github.Issue
-		mustHave    []string
-		mustNotHave []string
+		name             string
+		issue            *github.Issue
+		body             string
+		branch           string
+		secondIssue      *github.Issue
+		mustHave         []string
+		mustNotHave      []string
+		mustHaveComment1 bool
+		mustHaveComment2 bool
 	}{
 		{
 			name:        "LGTM with release-note",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNote}, true),
 			mustHave:    []string{lgtmLabel, releaseNote},
-			mustNotHave: []string{releaseNoteLabelNeeded},
+			mustNotHave: []string{releaseNoteLabelNeeded, doNotMergeLabel},
 		},
 		{
 			name:        "LGTM with release-note-none",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNoteNone}, true),
 			mustHave:    []string{lgtmLabel, releaseNoteNone},
-			mustNotHave: []string{releaseNoteLabelNeeded},
+			mustNotHave: []string{releaseNoteLabelNeeded, doNotMergeLabel},
 		},
 		{
 			name:        "LGTM with release-note-action-required",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNoteActionRequired}, true),
 			mustHave:    []string{lgtmLabel, releaseNoteActionRequired},
-			mustNotHave: []string{releaseNoteLabelNeeded},
+			mustNotHave: []string{releaseNoteLabelNeeded, doNotMergeLabel},
 		},
 		{
 			name:        "LGTM with release-note-experimental",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNoteExperimental}, true),
 			mustHave:    []string{lgtmLabel, releaseNoteExperimental},
-			mustNotHave: []string{releaseNoteLabelNeeded},
+			mustNotHave: []string{releaseNoteLabelNeeded, doNotMergeLabel},
 		},
 		{
 			name:        "LGTM with release-note-label-needed",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNoteLabelNeeded}, true),
-			mustHave:    []string{lgtmLabel, doNotMergeLabel, releaseNoteLabelNeeded},
-			mustNotHave: []string{},
+			mustHave:    []string{lgtmLabel, releaseNoteLabelNeeded},
+			mustNotHave: []string{doNotMergeLabel},
 		},
 		{
-			name:        "LGTM only",
-			issue:       github_test.Issue(botName, 1, []string{lgtmLabel}, true),
-			mustHave:    []string{lgtmLabel, doNotMergeLabel, releaseNoteLabelNeeded},
-			mustNotHave: []string{},
+			name:             "LGTM only",
```

Comment:
but we're not explicitly setting or removing it. I don't think it should be in the `mustNotHave` as a result, because there are other conditions (maybe not the munger) that may set it. `mustNotHave` is more for, we're explicitly removing it, so we need to verify it worked.